scripts/install.sh

```diff
@@ -22,6 +22,8 @@ elif [[ $ARCH == *386* ]] || [[ $ARCH == *686* ]]; then
   ARCH="i386"
 elif [[ $ARCH == *arm64* ]] || [[ $ARCH == *armv8* ]]; then
   ARCH="arm64"
+elif [[ $ARCH == *aarch64* ]] || [[ $ARCH == *armv8* ]]; then
```

Comment:
Can you remove the `|| [[ $ARCH == *armv8* ]]` part? It wont ever trigger because the line right above already grabs it. You could even just add ` || [[ $ARCH == *aarch64* ]]` to the line above.